authentication/src/main/kotlin/com/uber/sdk2/auth/api/request/PrefillInfo.kt

```diff
@@ -28,8 +29,8 @@ import kotlinx.parcelize.Parcelize
  */
 @Parcelize
 data class PrefillInfo(
-  val email: String,
-  val firstName: String,
-  val lastName: String,
-  val phoneNumber: String,
+  @Json(name = "email") val email: String,
```

Comment:
you probably don't need to annotate email since name is same as your variable

Reply:
yes, this is more like a future proof change in case someone changes the field name